app/views/layouts/autotune/application.html.erb

```diff
@@ -10,6 +10,15 @@
   <title>Autotune</title>
 
   <%=stylesheet_link_tag 'autotune/application', media: 'all' %>
+
+  <%=javascript_include_tag 'i18n' %>
+  <%=javascript_include_tag 'translations' %>
+  <script type="text/javascript">
+    I18n.defaultLocale = "<%= I18n.default_locale %>";
+    I18n.locale = "<%= I18n.locale %>";
+    I18n.fallbacks = true;
+  </script>
```

Comment:
Right now I'm not sure I follow how this might break something else but feel free to nudge me in the correct direction.